assets/js/payment-method-extensions/payment-methods/stripe/credit-card/use-payment-processing.js

```diff
@@ -92,7 +92,9 @@ export const usePaymentProcessing = (
 								paymentMethod: PAYMENT_METHOD_NAME,
 								paymentRequestType: 'cc',
 								stripe_source: sourceId,
-								shouldSavePayment,
+								// I think this is a saved token, so `shouldSavePayment`
+								// is redundant.
+								// 'wc-stripe-new-payment-method': shouldSavePayment,
```

Comment:
I _think_ we can delete all this, not relevant when paying with a saved payment method token.

Reply:
Two things:

- comment blocks are unnecessary - let's remove them.
- we'll still want to include the `wc-stripe-new-payment-method` property and value pair for this object. The reason why this block is here is in case there was an error needing handled during payment processing, the payment method keeps the already created source_id to avoid creating it again.